### PR TITLE
BackTop-translation

### PR DESCRIPTION
**Pull request by ddcat1115**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Rebase before creating a PR to keep commit history clear.
- [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by ddcat1115** [comment]
#1471

File: components/back-top/demo/custom.md [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+  en-US: Custom style
 ---
 
+## zh-CN
+
 可以自定义置顶按钮的样式，限制宽高：`40px * 40px`。
 
+## en-US
+
+You can customize the style of the button, just be aware of the size limit: no more than `40px * 40px`.
```
Comment (benjycui) [inline components/back-top/demo/custom.md]: be aware of  是意识到。。。

> ..., but please note that the max height/width of `BackTop` should not be greater thatn `40px`.

File: components/back-top/index.en-US.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## When To Use
+
+- When the page content is very long.
+- When you need to go back to the top very frequently in order to view the contents.
+
+## API
+
+> The distance to the bottom is set to `50px` by default, which is overridable.
+
+> If you deside to use custom styles, please be aware of the size limit: no more than `40px * 40px`.
```
Comment (benjycui) [inline components/back-top/index.en-US.md]: deside => decide

后半句是同样的问题。。

**Comment by afc163** [comment]
rebase 一下。。

**Pull request merged**